go/test/endtoend/cluster/vtgate_process.go

```diff
@@ -226,3 +227,21 @@ func VtgateProcessInstance(port int, grpcPort int, mySQLServerPort int, cell str
 
 	return vtgate
 }
+
+// GetVars returns map of vars
+func (vtgate *VtgateProcess) GetVars() map[string]interface{} {
+	resultMap := make(map[string]interface{})
+	resp, err := http.Get(vtgate.VerifyURL)
+	if err != nil {
+		return nil
```

Comment:
Should atleast log the error as we are not throwing it.

Reply:
Done. Refer PR[https://github.com/planetscale/vitess/pull/46]